Documentation/supported-mod-list.md

```diff
@@ -135,7 +142,7 @@ For mappacks - place *.bsp files to **valve/maps** folder, *.wad files to **valv
 90. [Half-Life C.A.G.E.D.](https://store.steampowered.com/app/679990/HalfLife_Caged/) (set **-num edicts 4096** in command-line parameters for successful start)
 90. [Half-Life: Dreamcast v1.1](https://www.moddb.com/mods/half-life-dreamcast)
 91. [Half-Life: OPS](https://www.runthinkshootlive.com/posts/half-life-ops/) 
-92. [Half-Life: Uplink](https://www.fileplanet.com/10360/10000/fileinfo/Half-Life:-UpLink-Demo)
+92. [Half-Life: Uplink](https://www.halflifeuplink.com) (download [hlulsl-2.1.zip](https://www.halflifeuplink.com/pub/hlulsl/hlulsl-2.1.zip), extract, rename `src` to `uplink` and move to base mod directory)
```

Comment:
For what purpose?
There already 4 links for different versions of this mod as you can see:
https://github.com/FWGS/xash3d-fwgs/blob/master/Documentation/supported-mod-list.md?plain=1#L273
https://github.com/FWGS/xash3d-fwgs/blob/master/Documentation/supported-mod-list.md?plain=1#L274

Also, the latest steam version already contains uplink files from further data.

Reply:
I don't think it really make sense to list `uplink` here at all, since it's shipped with the Steam version of Half-Life now. And we even check for `hldemo1.bsp` existence to show the checkbox in the menu now.

Other than that, clarifying how to run mods (once again) is fine.

Reply:
I think need to add separate file in Documentation with instruction how to load first Uplink map in 25th update.

This list originally was written for [old engine](https://github.com/FWGS/xash3d/wiki/List-of-mods-which-work-on-Android-and-other-non-Windows-platforms-without-troubles) using mod list by @Qwertyus3D from [moddb](https://www.moddb.com/engines/xash3d-engine/features/list-of-tested-half-life-singleplayer-mods).
Currently, need to fix broken links somehow and syncronize with fresh @Qwertyus3D's lists anyway.
And just actualize this list for new engine.

Reply:
I undid this change.